### PR TITLE
Enable MSTest runner in MSTest templates

### LINKED ISSUE #340: Update template to allow MSTest runner
With MSTest 3.2 we are introducing a new runner, it would be a good idea to update the template (with some flag) to allow creating a MSTest project with the runner

### PR DESCRIPTION
**Pull request by Evangelink**
Fixes #340

We will need to update to latest stable early next week

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -42,6 +42,12 @@
       "defaultValue": "false",
       "description": "Whether or not to enable packaging (via \"dotnet pack\") for the project."
     },
+    "EnableRunner": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner."
```
Comment (nohwnd) [inline template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-CSharp/.template.config/template.json:49]:
```suggestion
      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner"
```

Not grammatically correct, but makes copying the link easier if you can't simply click it in your commandline.

File: template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-VisualBasic/.template.config/template.json [hunk 2]
```diff
@@ -42,6 +42,12 @@
       "defaultValue": "false",
       "description": "Whether or not to enable packaging (via \"dotnet pack\") for the project."
     },
+    "EnableRunner": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner."
```
Comment (nohwnd) [inline template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-VisualBasic/.template.config/template.json:49]:
```suggestion
      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner"
```
same

File: template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-FSharp/.template.config/template.json [hunk 3]
```diff
@@ -42,6 +42,12 @@
       "defaultValue": "false",
       "description": "Whether or not to enable packaging (via \"dotnet pack\") for the project."
     },
+    "EnableRunner": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner."
```
Comment (nohwnd) [inline template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-FSharp/.template.config/template.json:49]:
```suggestion
      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner"
```

same

**Comment by Evangelink** [comment]
Do not merge, there is an issue for F#, I need to understand how to not include Program.fs in case of the runner.

**Review by cvpoienaru — DISMISSED** [review]

**Review by MarcoRossignoli — DISMISSED** [review]

**Pull request merged**